Unix/configeditor/configeditor.cpp

```diff
@@ -423,13 +446,30 @@ int MI_MAIN_CALL main(int argc, const char** argv)
             }
             ss << commentByte << prop << "=" << _CreateStringFromList(valueString,valueList);
         }
+        // Support querying a value from a property list
+        else if (!opts.query.empty())
+        {
+        	std::list<std::string> valueList;
+        	
+        	_ParseValueList(valueList, value);
+        	if (_IsValueInList(valueList, opts.query) != valueList.end())
+        	{
+        	    cout << prop << " have the value " << opts.query << std::endl;
+        	    return 0;
+        	}
+                cout << prop << " does not have the value " << opts.query << std::endl;
```

Comment:
Indentation for cout is off by one.

What sort of editor are you using where this sort of thing happens?